src/components/Feeds/FeedTags/FeedTagsElements.jsx

```diff
@@ -45,6 +45,7 @@ export const Tag = styled.p`
     word-break: break-word;
     overflow-wrap: break-word;
     text-transform: capitalize;
+    user-select: none;
```

Comment:
Wow, I learned a new css property that I didn't know that was exist haha.

Reply:
It's great that you discovered a new property. By the way, I've noticed that many components in the project could benefit from this property too.

Reply:
@kabir0x23